src/module/helpers-macros.js

```diff
@@ -87,3 +99,21 @@ export function rollItemMacro(itemName) {
   // Trigger the item roll
   return item.roll();
 }
+
+/**
+ * Roll on a RollTable by uuid if it exists.
+ *
+ * @param {string} tableUuId - UuId of the RollTable
+ */
+export function rollTableMacro(tableUuId) {
+  fromUuid(tableUuId).then((table) => {
+    if (table === null) {
+      return ui.notifications.error(
+        game.i18n.format("OSE.error.noRollTableWithUuId", {
+          uuid: tableUuId,
+        })
+      );
+    }
+    return table.draw({ roll: true, displayChat: true });
```

Comment:
`roll` option is expecting a Roll object

https://foundryvtt.com/api/classes/client.RollTable.html#draw

It looks like this still functions as implemented, perhaps ignoring passed options and using defaults when passed an option of the improper type.

since we are only using the default behavior, maybe we can remove the options? That way if Foundry decides to be strict we don't error out